src/components/flex/Flex.jsx

```diff
@@ -0,0 +1,265 @@
+// @flow
+
+import * as React from 'react';
+import type {Node} from 'react';
+import classNames from 'classnames';
+import {
+  FLEX_DIRECTION,
+  FLEX_JUSTIFY_VALUES,
+  FLEX_ALIGNMENT_VALUES,
+  SPACING_SET
+} from './FlexConsts';
+
+type FlexDirectionType =
+  | 'column'
+  | 'column-reverse'
+  | 'row'
+  | 'row-reverse';
+
+type FlexJustifyValuesType =
+  | 'center'
+  | 'flex-start'
+  | 'flex-end'
+  | 'baseline'
+  | 'space-between'
+  | 'space-around'
+  | 'space-evently'
+  | 'stretch';
+
+type FlexAlignmentValuesType =
+  | 'center'
+  | 'flex-start'
+  | 'flex-end'
+  | 'baseline'
+  | 'stretch';
+
+type FlexMarginsType =
+  | 'xs'
+  | 's'
+  | 'm'
+  | 'l'
+  | 'xl'
+  | 'xxl'
+  | 'xxxl'
+  | 'xxxxl';
+
+export {
+  FLEX_DIRECTION,
+  FLEX_JUSTIFY_VALUES,
+  FLEX_ALIGNMENT_VALUES,
+  SPACING_SET as FLEX_MARGINS
+};
+
+export type FlexPropsType = {
+  /**
+   * Children to be render inside of the Box. Takes no effect if **imgSrc** prop specified.
+   * @example <Flex>
+   *            children to be render here
+   *          </Flex>
+   *
+   */
+  children: ?Node,
```

Comment:
is children optional? if yes, it should be `children?: Node`, otherwise just `children: Node`. `Node` type is nullable by default, so `?Node` is the same as `Node`

Reply:
@PinkaminaDianePie I guess children should not be optional in this case. on my way. thx.